src/schema/document.rs

```diff
@@ -82,11 +82,16 @@ impl Document {
         self.add(FieldValue::new(field, Value::U64(value)));
     }
 
-    /// Add a u64 field
+    /// Add a i64 field
     pub fn add_i64(&mut self, field: Field, value: i64) {
         self.add(FieldValue::new(field, Value::I64(value)));
     }
 
+    /// Add a date field
+    pub fn add_date(&mut self, field: Field, value: i64) {
```

Comment:
Internally we want to handle it as a `i64`, but the API could expose something nicer to the user. 

Ideally that would have been a `DateTime` , but that's in the `chrono` trait, and it is generally not a good idea to use object in libraries within our library (it can lead to dependency hell). Let's ask for advise on the rust user forum what is the best practise for that.